scripts/RT/magic.R

```diff
@@ -77,6 +82,14 @@ WEPPout$Precip_mm<-WEPPout$Precip_in*25.4
 WEPPout$IrrDpth_mm<-WEPPout$IrrDpth_in*25.4
 WEPPout$SnwMeltWater_mm<-WEPPout$SnwMeltWater_in*25.4
 
+#****DEP management information
+m_colnames<-c("year","day","mgmt_flag","operation","details1","op2","details2")
+# set m_colwidths vector to read in management file ("Mgmt_0.txt") as a fixed width file
+widths<-c(4,4,2,21,14,17,11) #4 columns for year, 4 day, 2 mgmt flag, 21 mgmt file name, 14 mgmt file specifics, 17 2nd op, 11 2nd specifics
+widths<-as.integer(widths) #changes from numeric to integer
+WEPProt<- read_fwf(dir5, col_positions = fwf_widths(widths, col_names = m_colnames),

```

Comment:
The reference to "dir5" on line 90 I think should should be "weppmgmt" ("dir5" only existed on my computer)